src/videotile/VideoTileStatusCode.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+enum VideoTileStatusCode {
+  VideoEnded,
+  VideoRemovedDueToPolicy,
```

Comment:
I just noticed that the `VideoTileStatusCode` type is referenced in the observer, but PR doesn't use the enum value.

```
videoTileWasRemoved?(tileId: number, status?: VideoTileStatusCode): void;
```

Please remove an unused enum.

Reply:
Ok, I am removing the unused pieces, I will add them when I will iterate on other methods addition for simulcast notifications after this PR gets merged. Thanks!